tests/unit/neptune/new/internal/queue/test_disk_queue.py

```diff
@@ -0,0 +1,250 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import random
+import threading
+from dataclasses import dataclass
+from glob import glob
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import mock
+
+from neptune.internal.queue.disk_queue import (
+    DiskQueue,
+    QueueElement,
+)
+
+
+def test_put():
+    with TemporaryDirectory() as dir_path:
+        with DiskQueue[Obj](
+            dir_path=Path(dir_path),
+            to_dict=serializer,
+            from_dict=deserializer,
+            lock=threading.RLock(),
+        ) as queue:
+            # given
+            obj = Obj(5, "test")
+            queue.put(obj)
+
+            # when
+            queue.flush()
+
+            # then
+            assert queue.get() == get_queue_element(obj, 1)
```

Comment:
Something that rarely comes up but bugs me: "expected" should be on the left, "actual" on the right.

At least that should be the case according to the message displayed when the test fails - it labels the expression on the left as "expected:" and the other as "actual:". I already wasted 15 minutes trusting these labels instead of triple-checking what each statement in a test assert means.

Reply:
You're 100% right. Will change that in a moment.

Reply:
Done